src/utils/validateProjectName.js

```diff
@@ -0,0 +1,39 @@
+const defaultErrorMessages = {
+  MIN_8: 'Min 8 characters',
+  MIN_2_CHAR: 'Min 2 characters in sequence',
+  INVALID_CHARS: 'Invalid characters',
+  NAME_EXISTS: 'A project with the same name exists',
```

Comment:
are we going to display this text to the user upon error in validation? If that will be the case, all of these apart from `NAME_EXISTS` will need rephrasing to be clearer

Reply:
Yes, but it couldn't be too long because it's used in tight spaces. Do you have any suggestions to improve it?

Reply:
Where do we use it? Maybe you can send me a screenshot?

Reply:
I will actually take a look at staging

Reply:
yes, I took a look, it looks fine as it is